packages/turf-line-arc/test.js

```diff
@@ -24,12 +24,54 @@ test("turf-line-arc", (t) => {
   for (const { filename, name, geojson } of fixtures) {
     const { radius, bearing1, bearing2, steps, units } = geojson.properties;
     const arc = truncate(
-      lineArc(geojson, radius, bearing1, bearing2, steps, units)
+      lineArc(geojson, radius, bearing1, bearing2, { steps, units })
     );
     const results = featureCollection([geojson, arc]);
-
     if (process.env.REGEN) write.sync(directories.out + filename, results);
     t.deepEquals(results, load.sync(directories.out + filename), name);
+    // Check the resulting arc geometry has the expected number of points.
+    // undefined or 0 steps should default to 64 + 1 points, 1 to 1 + 1,
+    // 2 to 2 + 1, ...
+    const expectedPoints = geojson.properties?.steps
+      ? geojson.properties?.steps + 1
+      : 64 + 1;
```

Comment:
WIthout any context seeing that the arc would get N+1 points feels weird. The jsdoc on the function doesn't indicate that it would be +1 either.

Reply:
I think you can 'understand' steps as either the number of points or the number of segments. It isn't documented so I suppose we're free to do whichever we want (and maybe document it?). We could maybe consider the previous behavior as the expected behavior. I'm not really tied to that as being important, but maybe we should document whatever the outcome is in the jsdoc at least.

Reply:
I'll push a commit updating the docs, and then merge. Thanks for reviewing 👍